src/modules/pages/components/RouteLayouts/Default/Default.tsx

```diff
@@ -45,10 +51,24 @@ const Default = ({
           />
         )}
         <div className={styles.content}>
-          {hasSubscribedColonies && (
-            <div className={styles.coloniesList}>
-              <SubscribedColoniesList />
+          {isMobile ? (
+            // Render the UserNavigation and SubscribedColoniesList in shared parent on mobile
+            <div className={styles.head}>
+              <div className={navStyles.nav}>
+                <UserNavigation />
+              </div>
+              {hasSubscribedColonies && (
+                <div className={styles.coloniesList}>
+                  <SubscribedColoniesList />
+                </div>
+              )}
             </div>
+          ) : (
+            hasSubscribedColonies && (
+              <div className={styles.coloniesList}>
+                <SubscribedColoniesList />
+              </div>
+            )
```

Comment:
Please try to reduce code repetition as much as possible:
```suggestion
const SubscribedColonies = () => hasSubscribedColonies ? (
    <div className={styles.coloniesList}>
      <SubscribedColoniesList />
    </div>
  ) : null;

{isMobile ? (
  // Render the UserNavigation and SubscribedColoniesList in shared parent on mobile
  <div className={styles.head}>
    <div className={navStyles.nav}>
      <UserNavigation />
    </div>
    {SubscribedColonies()}
  </div>
) : SubscribedColonies()
```